backend/tracim_backend/config.py

```diff
@@ -227,10 +227,24 @@ def __init__(self, settings: typing.Dict[str, typing.Any]):
             'user.auth_token.validity',
             '604800',
         ))
-        self.USER_RESET_PASSWORD_TOKEN_VALIDITY = int(settings.get(
-            'user.reset_password.validity',
-            '900'
-        ))
+
+        # TODO - G.M - 2019-03-14 - retrocompat code,
+        # will be deleted in the future (https://github.com/tracim/tracim/issues/1483)
+        legacy_reset_password_validity = settings.get('user.reset_password.validity', None)
+        if legacy_reset_password_validity:
+            logger.warning(
+                self,
+                'user.reset_password.validity parameter is deprecated ! '
+                'please use user.reset_password.token_validity instead.'
+            )
+        reset_password_validity = settings.get(
+            'user.reset_password.token_validity',
+            None
+        )
+        defaut_reset_password_validity = '900'
+        self.USER_RESET_PASSWORD_TOKEN_VALIDITY = int(reset_password_validity or \
```

Comment:
`USER_RESET_PASSWORD_TOKEN_VALIDITY` => `USER_RESET_PASSWORD_TOKEN_LIFETIME`

Reply:
same

Reply:
I prefer that also.

Reply:
done